transaction/core/src/ring_signature/mlsag.rs

```diff
@@ -347,11 +364,24 @@ impl RingMLSAG {
     }
 }
 
+fn decompress_ring(
```

Comment:
This should now be the only place that input rings are decompressed on the consensus server.